### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
# Feature added : 

- [x] Made custom hook to get notes.

### CODE REVIEW

**Comment by AtulPant2704** [comment]
Sidebar is not there in your components folder because of which UI is breaking, add that component.

File: src/hooks/useNotes.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { useAuth } from './../context/auth/authContext';
+export function useNotes() {
+  const { user } = useAuth();
+  const [notes, setNotes] = useState([]);
+  useEffect(() => {
+    const getProducts = async () => {
+      try {
+        const response = await axios.get('/api/notes', {
+          headers: {
+            authorization: user.token,
```
Comment (AtulPant2704) [inline src/hooks/useNotes.js:12]: In LMS the requested URL mentioned for getting notes is :-
```
GET /api/notes[#](https://mockbee.netlify.app/docs/api/apps/notes-app#1-get-apinotes)
Request URL: /api/notes
```
I think you don't need to pass token to it.

File: src/hooks/useNotes.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+          },
+        });
+        console.log(response.data.notes);
```
Comment (AtulPant2704) [inline src/hooks/useNotes.js]: You can remove this console.log

**Review by AtulPant2704 — APPROVED** [review]
Other then the one thing mentioned, rest all looks fine.

File: src/hooks/useNotes.js (hunk 1 above)
Comment (AtulPant2704) [reply src/hooks/useNotes.js:12]: Oh sorry, my bad, you do need the encoded token.

Comment (Kevin-Solomon) [reply src/hooks/useNotes.js:12]: We need the token to get user notes.

**Review by mohammed-sas — APPROVED** [review]
Code looks clean
Functionalities are working as intended

**Pull request merged**